### DISCUSSION TITLE
Automatic  Postlight Parser for specific feeds

### DISCUSSION THREAD

**Discussion by jpsnyder** [jocmp/capyreader#207] [Feature Requests]
In feedbin, there is a feature where you can have it remember what feeds you selected to extract full content to replace partial articles. I think that option was called "Sticky Full Content".

I know you have the ability to have it pull the full content for all articles, but I was wondering if you had plans to make it just do that for feeds where I have selected that option manually?

Loving the app btw!

**Comment by jocmp** [comment]
hey @jpsnyder, glad you're enjoying the app! I don't have plans right now to make it per-feed so that I can focus on other features, but I'd like to add it in future.

One of my inspirations is the [Reeder](https://reederapp.com) Mac app which is highly configurable but super simple, so if I can add it in a similar way per-feed that's what I want to do.

I'll track this via https://github.com/jocmp/capyreader/issues/215

**Reply by jpsnyder** [reply]
Okay cool. No worries on time. I'm just glad it is on the radar. 

The way feedbin does it, is that if you turn on the parser for an article, it simply remembers you had it on, and keeps it on for any article of that feed. To turn it off, simple click the icon again within an article to turn off the parser, and it will remember to have it off for all articles of that feed. Therefore, the only UI change would be to have a toggle option in the settings to enable or disable this feature.

**Reply by jocmp** [reply]
aha, I see the difference now. I tried it out with my Feedbin account too. This is what I meant to implement the first time around.

**Reply by jocmp** [reply]
I took a swing at adding this feature. I'll include it in the next release. Here's a demo of what it looks like: each feed is only stuck when it's selected. It becomes unstuck when you hit the full content button or toggle it off in settings.

https://github.com/user-attachments/assets/ed23376f-b564-42b3-be4e-0dc9c1915fae

**Reply by jpsnyder** [reply]
That's awesome! Exactly what I was hoping for. Thanks. It now feels more feature complete as a Feedbin client.

**Comment by jocmp** [comment]
I did find a bug in 2024.07.1019 where refreshing a feed will reset it's setting. I have a fix I'll include in the next release (https://github.com/jocmp/capyreader/pull/226).
